lib/open_mic.rb

```diff
@@ -0,0 +1,24 @@
+class OpenMic
+
+  attr_reader :location, :date, :performers
+
+  def initialize(attributes)
+    @location = attributes[:location]
+    @date = attributes[:date]
+    @performers = []
+  end
+
+  def welcome(user)
+    @performers << user
+  end
+
+  def repeated_jokes?
+    jokes1 = @performers[0].jokes
+    jokes2 = @performers[1].jokes
+    if jokes1.include?(jokes2[0]) || jokes1.include?(jokes2[1])
```

Comment:
I like this approach, but it would need to be more robust to handle a large number of performers and/or jokes. Can you think of any enumerables that could help handle larger sets of data?